packages/framework/client-logger/external-telemetry/package.json

```diff
@@ -0,0 +1,76 @@
+{
+	"name": "@fluidframework/external-telemetry",
+	"version": "2.0.0-rc.3.0.0",
+	"description": "External Fluid telemetry types and classes for both creating consuming said telemetry",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/client-logger/external-telemetry"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "dist/index.js",
+	"module": "lib/index.mjs",
+	"types": "dist/index.d.ts",
+	"scripts": {
+		"api": "fluid-build . --task api",
+		"api-extractor:commonjs": "api-extractor run --local",
+		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
+		"build": "fluid-build . --task build",
+		"build:commonjs": "fluid-build . --task commonjs",
+		"build:compile": "fluid-build . --task compile",
+		"build:docs": "fluid-build . --task api",
+		"build:esnext": "tsc-multi --config ../../../../common/build/build-common/tsc-multi.esm.json",
```

Comment:
We've moved away from tsc-multi, there's a newer pattern for doing both CJS and ESM builds. I can add those changes to this branch, so you can focus on the functional aspect.

Reply:
I've updated the branch to get away from tsc-multi. I appreciate the help